### PR TITLE
[EventGrid] Add system classes for Azure Communication Services

### PR DESCRIPTION
**Pull request by YijunXieMS**
[Swagger file](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json) of azure communication services was added to EventGrid.

This PR is to add the types defined in the swagger file to EventGrid java code. 

---update 09/23/2020---
Happened to find that there is no mapping for KeyVault, Web and MachineLearning events. Added to this PR.

### CODE REVIEW

**Review by lmazuel — APPROVED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventMappings.java [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
+import com.azure.messaging.eventgrid.systemevents.ACSChatMemberAddedToThreadWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMemberRemovedFromThreadWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageDeletedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageEditedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageReceivedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadCreatedWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadPropertiesUpdatedPerUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadWithUserDeletedEventData;
+import com.azure.messaging.eventgrid.systemevents.AcssmsDeliveryReportReceivedEventData;
+import com.azure.messaging.eventgrid.systemevents.AcssmsReceivedEventData;
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventMappings.java:15]: It's odd that some classes use Pascal case and some use all uppercase letters for ACS. Can we use Pascal case throughout?

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcssmsReceivedEventData.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+package com.azure.messaging.eventgrid.systemevents;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.OffsetDateTime;
+
+/** The AcssmsReceivedEventData model. */
+@Fluent
+public final class AcssmsReceivedEventData extends AcssmsEventBaseProperties {
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcssmsReceivedEventData.java:13]: The word boundary was not correctly interpreted by AutoRest. This should have bee `AcsSmsReceivedEventData`. You can use directives in autorest to rename these models.

**Review by srnagar — APPROVED** [review]
LGTM!

**Pull request merged**